docs/getting-started-guides/digitalocean.md

```diff
@@ -36,24 +36,33 @@ As a minimum you will need to set these environment variables -
 
 ```
 # v2 API token
-DO_API_TOKEN=
-```
 
+TF_VAR_do_token=
+
+```
 To generate a v2 API token see [https://www.digitalocean.com/community/tutorials/how-to-use-the-digitalocea n-api-v2](https://www.digitalocean.com/community/tutorials/how-to-use-the-digitalocean-api-v2)
 
 ```
+# v1 API credentials as needed by ansible dynamic inventory.
```

Comment:
dont think you resolved your conflicts correctly, because we removed the v1 API docs in https://github.com/Capgemini/Apollo/commit/9fcf6ee630f611e7d36ed65c939a6adb1e195f56